.github/workflows/main.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          fetch-depth: 0
+          fetch-depth: 1
```

Comment:
Maybe useless, because the default value is 1.
source https://github.com/actions/checkout